README.md

```diff
@@ -23,15 +23,22 @@ EOF
 make
 
 # Set the environment variables
-export DRIVER_IMAGE=registry.k8s.io/csi-secrets-store/driver:v1.3.3
 export OPERATOR_NAME=secrets-store-csi-driver-operator
-export PROVISIONER_IMAGE=quay.io/openshift/origin-csi-external-provisioner:latest
-export RESIZER_IMAGE=quay.io/openshift/origin-csi-external-resizer:latest
-export SNAPSHOTTER_IMAGE=quay.io/openshift/origin-csi-external-snapshotter:latest
+export DRIVER_IMAGE=registry.k8s.io/csi-secrets-store/driver:v1.3.3
```

Comment:
Would it be better to have
```
export DRIVER_IMAGE=quay.io/openshift/origin-secrets-store-csi-driver:latest
```
there?

Reply:
I tried this, but the pods failed to start with:
```
        message: Back-off pulling image "quay.io/openshift/origin-secrets-store-csi-driver:latest"  
        reason: ImagePullBackOff
```
So I think we should leave this as-is until [ART-7070](https://issues.redhat.com/browse/ART-7070) is complete and the first published image is available.